_opensearch/query-dsl/index.md

```diff
@@ -12,9 +12,31 @@ redirect_from:
 
 # Query DSL
 
-While you can use HTTP request parameters to perform simple searches, you can also use the OpenSearch query domain-specific language (DSL), which provides a wider range of search options. The query DSL uses the HTTP request body, so you can more easily customize your queries to get the exact results that you want.
+OpenSearch provides a query domain-specific language (DSL) that you can use to search with more options than a simple search via HTTP request parameter alone. The query DSL uses the HTTP request body, so you can more easily customize your queries to get the exact results that you want.
 
-For example, the following request performs a simple search to search for a `speaker` field that has a value of `queen`.
+The OpenSearch query DSL provides three query options: term-level queries, full-text queries, and boolean queries. You can even perform more complicated searches by using different elements from each variety to find whatever data you need.
+
+## DSL Query Types
+
+OpenSearch supports two types of queries when you search for data: term-level queries and full-text queries.
+
+The following table describes the differences between them:
+
+| Metrics | Term-level queries | Full-text queries
+:--- | :--- | :---
+*Query results* | Term-level queries answer which documents match a query. | Full-text queries answer how well the documents match a query.
+*Analyzer* | The search term isn't analyzed. This means that the term query searches for your search term as it is.  | The search term is analyzed by the same analyzer that was used for the specific field of the document at the time it was indexed. This means that your search term goes through the same analysis process that the document's field did.
```

Comment:
Last sentence: would this be better as "This means that your search term goes through the same analysis process as the document's field"?

Reply:
thanks